L1Trigger/L1TGlobal/plugins/TriggerMenuParser.h

```diff
@@ -425,6 +425,9 @@ namespace l1t {
     std::vector<std::vector<CaloTemplate> > m_corCaloTemplate;
     std::vector<std::vector<EnergySumTemplate> > m_corEnergySumTemplate;
 
+    /// AXOL1TL Model Version
+    std::string m_axol1tlModelVersion = "";
```

Comment:
will be removed (commit ready, waiting for tests to finish)

Reply:
is there a reason to let tests to finish?